test/components/views/elements/LabelledCheckbox-test.tsx

```diff
@@ -30,32 +28,18 @@ jest.mock("matrix-js-sdk/src/randomstring", () => {
 
 describe("<LabelledCheckbox />", () => {
     type CompProps = React.ComponentProps<typeof LabelledCheckbox>;
-    const getComponent = (props: CompProps) => mount(<LabelledCheckbox {...props} />);
-    type CompClass = ReturnType<typeof getComponent>;
+    const getComponent = (props: CompProps) => <LabelledCheckbox {...props} />;
+    const getCheckbox = (): HTMLInputElement => screen.getByRole("checkbox");
 
-    const getCheckbox = (component: CompClass) => component.find(`input[type="checkbox"]`);
-    const getLabel = (component: CompClass) => component.find(`.mx_LabelledCheckbox_label`);
-    const getByline = (component: CompClass) => component.find(`.mx_LabelledCheckbox_byline`);
-
-    const isChecked = (checkbox: ReturnType<typeof getCheckbox>) => checkbox.is(`[checked=true]`);
-    const isDisabled = (checkbox: ReturnType<typeof getCheckbox>) => checkbox.is(`[disabled=true]`);
-    const getText = (span: ReturnType<typeof getLabel>) => (span.length > 0 ? span.at(0).text() : null);
-
-    test.each([null, "this is a byline"])("should render with byline of %p", (byline) => {
+    it.each([null, "this is a byline"])("should render with byline of %p", (byline) => {
         const props: CompProps = {
             label: "Hello world",
             value: true,
             byline: byline,
             onChange: jest.fn(),
         };
-        const component = getComponent(props);
-        const checkbox = getCheckbox(component);
-
-        expect(component).toMatchSnapshot();
-        expect(isChecked(checkbox)).toBe(true);
```

Comment:
If we compare with a snapshot, all these expects for „being checked“ etc. are not necessary.